invenio_oauthclient/contrib/github.py

```diff
@@ -118,8 +118,8 @@ def account_info(remote, resp):
     ghuser = gh.me()
     email = ghuser.email
     if not email:
-        record = next(gh.iter_emails(1))
-        email = record['email'] if 'email' in record else None
+        record = next(gh.emails())
```

Comment:
- You should be able to sign in with a github user, logout, change github username, and login again.
- If no primary and verified email exists, then ask user to provide email address.